.github/workflows/test.yml

```diff
@@ -41,25 +42,36 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: npm
       - run: npm clean-install
-      - name: Ensure dependencies are compatible with the version of node
-        run: npx ls-engines
-      - run: npm run test:ci
+      - run: npm audit signatures
+      - run: npm test
 
-  # separate job to set as required in branch protection,
-  # as the build names above change each time Node versions change
-  test:
+  # verify against the node version defined for development in the .nvmrc
+  test_dev:
     runs-on: ubuntu-latest
-    needs: test_matrix
-    if: always()
+    timeout-minutes: 5
+
     steps:
       - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4
-      - uses: actions/setup-node@5e21ff4d9bc1a8cf6de233a3057d20ec6b3fb69d # v3
+      - run: git config --global user.name github-actions
+      - run: git config --global user.email github-actions@github.com
+      - name: Use Node.js from .nvmrc
+        uses: actions/setup-node@5e21ff4d9bc1a8cf6de233a3057d20ec6b3fb69d # v3
         with:
-          node-version: lts/*
+          node-version-file: .nvmrc
           cache: npm
       - run: npm clean-install
       - run: npm audit signatures
-      - run: npm run lint
```

Comment:
since this job is now specifically about summarizing the verification results from the `test_dev` and `test_matrix` jobs, it doesnt need to checkout, setup node, or run additional verification